src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

```diff
@@ -160,6 +158,25 @@ Other ways to create and manage keys:
   </tbody>
 </table>
 
+</Collapser>
+</CollapserGroup>
+
+## View and manage API keys [#keys-ui]
+
+You can view and manage most API keys from the [API keys UI page](https://one.newrelic.com/api-keys) by clicking **API keys** from the [account dropdown](/docs/glossary/glossary/#account-dropdown). You can also:
```

Comment:
```suggestion
You can view and manage most API keys from the [API keys UI page](https://one.newrelic.com/api-keys) by clicking **API keys** from the [account dropdown](/docs/glossary/glossary/#account-dropdown) menu. You can also:
```

or maybe delete 'the' so it just sounds 'from account dropdown'

Reply:
Good catch!